dcurve/src/internal/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { GrantRoundContributions, Contribution, GrantsDistribution, GrantMatch } from '../../src/types';
+
+/**
+ * util function which adds anon contribution of given value to the
+ * grantRoundContributions for given grant
+ *
+ * @param grantId
+ * @param grantRoundContributions
+ * @param amount
+ * @returns grantRoundContributions
+ */
+export const addAnonContribution = (
```

Comment:
I don't fully understand the purpose of this method, but not a big deal since it seems this is a detail of the algorithm implementation

Reply:
Updating the description and renaming the method to make it cleaner